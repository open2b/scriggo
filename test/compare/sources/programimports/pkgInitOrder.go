// skip : declarations in imported package are not ordered https://github.com/open2b/scriggo/issues/254

// rundir 