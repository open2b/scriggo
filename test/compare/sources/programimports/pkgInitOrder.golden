200 100
53.122 43.122